src/modules/src/estimator_kalman.c

```diff
@@ -593,4 +599,52 @@ PARAM_GROUP_START(kalman)
  * @brief Nonzero to use robust TWR method (default: 0)
  */
   PARAM_ADD_CORE(PARAM_UINT8, robustTwr, &robustTwr)
+/**
+ * @brief Process noise for x and y acceleration
+ */
+  PARAM_ADD_CORE(PARAM_FLOAT, pNAcc_xy, &coreParams.procNoiseAcc_xy)
+/**
+ * @brief Process noise for z acceleration
+ */
+  PARAM_ADD_CORE(PARAM_FLOAT, pNAcc_z, &coreParams.procNoiseAcc_z)
+  /**
+ * @brief Process noise for velocity
+ */
+  PARAM_ADD_CORE(PARAM_FLOAT, pNVel, &coreParams.procNoiseVel)
+  /**
+ * @brief Process noise for position
+ */
+  PARAM_ADD_CORE(PARAM_FLOAT, pNPos, &coreParams.procNoisePos)
+  /**
+ * @brief Process noise for attitude
+ */
+  PARAM_ADD_CORE(PARAM_FLOAT, pNAtt, &coreParams.procNoiseAtt)
+  /**
+ * @brief Measurement noise for barometer
+ */
+  PARAM_ADD_CORE(PARAM_FLOAT, mNBaro, &coreParams.measNoiseBaro)
+  /**
+ * @brief Measurement Noise for roll/pitch gyros
+ */
+  PARAM_ADD_CORE(PARAM_FLOAT, mNGyro_rollpitch, &coreParams.measNoiseGyro_rollpitch)
+  /**
+ * @brief Measurement Noise for yaw gyro
+ */
+  PARAM_ADD_CORE(PARAM_FLOAT, mNGyro_yaw, &coreParams.measNoiseGyro_yaw)
```

Comment:
I realize this is copy-and-paste, but. Since you are here :) could we fix the spurious capital N in Noise here?

"Measurement Noise for roll/pitch gyros" => "Measurement noise for roll/pitch gyros"

etc.